### DISCUSSION TITLE
Default flipper instance not configured

### DISCUSSION THREAD

**Discussion by weslyl** [jnunemaker/flipper#534] [Q&A] [answered]
Hello! I am new to flipper and have been trying to integrate it with rake. I have been running into this error and I am not sure what steps are there to resolve it.

<img width="1415" alt="Screen Shot 2021-06-03 at 12 01 51 PM" src="https://user-images.githubusercontent.com/81185669/120707271-fa512280-c46e-11eb-86c2-3cf86732094d.png">

Thanks!

**Comment by jnunemaker** [comment]
What version of flipper is in your gemfile? 

`bundle show flipper`

On June 3, 2021, David Grevink ***@***.***> wrote:
> Hello! I am new to flipper and have been trying to integrate it with
> rake. I have been running into this error and I am not sure what steps
> are there to resolve it.
>
>
>

> Thanks!
>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/jnunemaker/flipper/discussions/534>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAAAB25XLP6STVNNJYHROZTTQ7QIDANCNFSM46BOSWSQ>.

**Reply by weslyl** [reply]
I am currently running flipper-0.16.2

**Comment by jnunemaker** [comment, marked as answer]
I’d recommend updating to the latest. Should be like 0.21. 

The issue you are having is that flipper isn’t configured at all. You’ll
need to pick an adapter and configure the default instance as the docs
recommend. 

If you update to the latest flipper version the default instance is
configured for you when you pick an adapter. 

On June 3, 2021, David Grevink ***@***.***> wrote:
> I am currently running flipper-0.16.2
>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/jnunemaker/flipper/discussions/534#discussioncomment-
823589>, or unsubscribe <https://github.com/notifications/unsubscribe-
auth/AAAAB27VCZXCQ7ZTAFXU573TQ747NANCNFSM46BOSWSQ>.
